compiler-rt/test/CMakeLists.txt

```diff
@@ -106,6 +106,9 @@ if(COMPILER_RT_CAN_EXECUTE_TESTS)
   # ShadowCallStack does not yet provide a runtime with compiler-rt, the tests
   # include their own minimal runtime
   add_subdirectory(shadowcallstack)
+  # PointerFieldProtection does not yet provide a runtime with compiler-rt, the tests
+  # include their own minimal runtime
```

Comment:
Remove this comment, we do have a runtime component (emupac) and "the tests include their own minimal runtime" isn't accurate either.